src/azul/indexer.py

```diff
@@ -159,6 +159,7 @@ def _create_indices(self):
                                      ignore=[400],
                                      body=dict(settings=self.settings(index_name),
                                                mappings=dict(doc=self.mapping())))
+            es_client.cluster.put_settings(body=dict(transient={'script.max_compilations_per_minute': 60}))
```

Comment:
> This method (_create_indices) is called a lot, once per bundle notification. Think of ways to avoid touching the cluster settings every time. If this is a cluster setting, should this go into TF config?
I was unable to find resources that modify the cluster settings via terraform.